app/templates/base.html

```diff
@@ -11,44 +11,242 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
     <link rel="stylesheet" href="{{ url_for('static', filename='styling-css/common-elements-style.css') }}">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
+    <link rel="stylesheet" href="{{ url_for('static', filename='styling-css/posts.css') }}">
     <script src="{{ url_for('static', filename='common-elements.js') }}"></script>
+
 </head>
 <body>
     <!-- The header section with application theme, logo and user profile options menu -->
-    <div class="header">
-        <!-- Community Logo in the header -->
-        <a href="#" class="AppLogo">
-            <img src="{{ url_for('static',
-            filename='images/AppLogo.jpg') }}" alt="ApplicationLogo">
-        </a> 
-        <!-- The name of application in the header -->
-        <p class="application-name">Local Community Board</p>
-        <!-- The user profile options menu has options to logout and view user profile details -->
-        <nav class="navigation-menu">
-            <div class="fa-solid fa-xmark" id="cancel-button"></div>
-            <div class="profile-content">
-                <h3>User Name</h3>
-                <p>User Id</p> 
+    <div class="page-container">
+        <div id="header-sidebar-container">
+            <div class="header">
+                <!-- Community Logo in the header -->
+                <a href="#" class="AppLogo">
+                    <img src="{{ url_for('static',
+                    filename='images/AppLogo.jpg') }}" alt="ApplicationLogo">
+                </a> 
+                <!-- The name of application in the header -->
+                <p class="application-name">Local Community Board</p>
+                <div class="form">
+                    <label for="search">
```

Comment:
This code should be there in index.html file. We don't want to have search functionality and create request option on all pages like view request, my profile, etc.